### PR TITLE
Add statutory instrument

### PR DESCRIPTION
**Pull request by elliotcm**
Part of https://github.com/alphagov/specialist-publisher/pull/1228

https://trello.com/c/mmvxwKWa/206-spike-create-new-type-in-specialist-publisher-to-handle-statutory-instruments
https://trello.com/c/6wjzkfut/207-spike-create-finder-for-new-statutory-instrument-specialist-document-type
https://trello.com/c/mxh2qAcv/220-drop-down-for-subjects-in-eu-exit-statutory-instrument-specialist-publisher-document-type
https://trello.com/c/XuhoYZak/221-date-field-for-laid-date-in-eu-exit-statutory-instrument-specialist-publisher-document-type

### CODE REVIEW

File: dist/formats/specialist_document/frontend/schema.json [hunk 1]
```diff
@@ -46,24 +46,25 @@
     "document_type": {
       "type": "string",
       "enum": [
+        "aaib_report",
+        "asylum_support_decision",
+        "business_finance_support_scheme",
         "cma_case",
-        "employment_appeal_tribunal_decision",
```
Comment (barrucadu) [inline dist/formats/specialist_document/frontend/schema.json]: There's a bunch of seemingly-unrelated changes here.  Is this an automatically generated file we've just not kept up to date?

Comment (thomasleese) [reply dist/formats/specialist_document/frontend/schema.json]: I think that's just making this list alphabetical?

**Comment by kevindew** [comment]
Be good to break up the last commit. There's a convention that you have a separate commit to rebuild the schemas so it's easier to review e.g: https://github.com/alphagov/govuk-content-schemas/pull/773/commits

**Comment by elliotcm** [comment]
@kevindew @barrucadu @thomasleese Yes all correct, I'll revisit that commit

**Comment by elliotcm** [comment]
@kevindew @barrucadu @thomasleese Updated.

File: formats/specialist_document.jsonnet [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-    "tax_tribunal_decision",
+    "medical_safety_alert",
     "raib_report",
-    "drug_safety_update",
-    "business_finance_support_scheme",
-    "asylum_support_decision",
     "residential_property_tribunal_decision",
+    "service_standard_report",
+    "tax_tribunal_decision",
+    "utaac_decision",
```
Comment (kevindew) [inline formats/specialist_document.jsonnet]: ❤️
